src/components/ToolbarMenu.js

```diff
@@ -14,27 +14,52 @@ import React, { PropTypes } from 'react';
  *
  * A menu style button to use within the Toolbar.
  */
-export function ToolbarMenu({ title, label, children }) {
-  return (
-    <a
-      className="toolbar-menu toolbar-button"
-      onMouseDown={preventDefault}
-      title={title}>
-      {label}
-      <svg width="14" height="8">
-        <path fill="#666" d="M 5 1.5 L 14 1.5 L 9.5 7 z" />
-      </svg>
-      <ul className="toolbar-menu-items">
-        {children}
-      </ul>
-    </a>
-  );
-}
+export class ToolbarMenu extends React.Component {
+  static propTypes = {
+    title: PropTypes.string,
+    label: PropTypes.string,
+  }
 
-ToolbarMenu.propTypes = {
-  title: PropTypes.string,
-  label: PropTypes.string,
-};
+  constructor(props) {
+    super(props);
+    this.state = { visibility: 'hidden' };
+  }
+
+  render() {
+    const ulStyle = {
+      visibility: this.state.visibility
```

Comment:
(Non-blocking feedback.)

I probably would have gone with a class here. It would result in a smaller change to the CSS file, and would mean we'd be more consistently using one method (predominantly class selectors) of applying presentation throughout the app.